src/session/SessionMiddleware.ts

```diff
@@ -28,37 +29,35 @@ function sessionRequestHandler(config: CookieConfig, sessionStore: SessionStore)
 
         if (sessionCookie) {
 
-            console.log("Got a session cookie.");
-            console.log(`REQUEST: ${request.url}`);
-            console.log(`COOKIE: ${sessionCookie}`);
+            loggerInstance().infoRequest(request,`Session cookie ${sessionCookie} found in request: ${request.url}`);
 
             try {
-                validateCookieSignature(sessionCookie, config.cookieSecret)
-
+                validateCookieSignature(sessionCookie, config.cookieSecret);
                 const cookie = Cookie.createFrom(sessionCookie);
                 const sessionData = await sessionStore.load(cookie);
                 const session = new Session(sessionData);
                 session.verify();
                 request.session = session;
+                loggerInstance().debug(`Session successfully loaded from cookie ${sessionCookie}`)
 
             } catch (err) {
 
-                console.error(err);
+                loggerInstance().error(`Session loading failed from cookie ${sessionCookie} due to error: ${err}`);
                 response.clearCookie(config.cookieName);
                 delete request.session;
 
+
                 try {
                     const cookie = Cookie.createFrom(sessionCookie);
                     sessionStore.delete(cookie);
-                } catch (_) {
-                    console.error(_);
+                } catch (err) {
+                    loggerInstance().error(err);
                 }
 
             }
 
         } else {
-            console.log(`REQUEST: ${request.url}`);
-            console.log("No Session cookie.");
+            loggerInstance().infoRequest(request,`Session cookie ${sessionCookie} not found in request ${request.url}`);
```

Comment:
I was bit silly here. If course `sessionCookie` will evaluate to undefined here so no  point printing it - apologies. Would you change log message to `Session cookie not found in request ${request.url}`